crates/bpe/README.md

```diff
@@ -193,4 +197,39 @@ We compared our implementations with the tiktoken implementation on a MacBook Pr
 As can be seen, our Backtracking implementation beats the TikToken Rust implementation by ~4x.
 And even the fully dynamic programming solution is faster with a more consistent runtime.
 The tuned heap implementation is still quite competitive to TikToken (especially for smaller inputs).
-If the requirement of correct BPE output can be relaxed, then the Greedy approach or the minimal encoding approach are the clear winners.
\ No newline at end of file
+If the requirement of correct BPE output can be relaxed, then the Greedy approach or the minimal encoding approach are the clear winners.
+
+### Counting results
```

Comment:
Maybe mention that we are not using the original tiktoken implementation, but some fork which provides a wrapper... (maybe it is easy to migrate to the original implementation now?)

Reply:
I've added a note about that in 537699192c9eff01f906e785f549f06eb88217af.